package.json

```diff
@@ -20,19 +20,15 @@
   "private": false,
   "dependencies": {
     "babel-plugin-react-transform": "^2.0.0",
-    "babel-plugin-syntax-flow": "^6.13.0",
     "babel-plugin-syntax-trailing-function-commas": "^6.5.0",
     "babel-plugin-transform-async-to-generator": "^6.8.0",
     "babel-plugin-transform-class-properties": "^6.4.0",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-export-extensions": "^6.4.0",
-    "babel-plugin-transform-flow-strip-types": "^6.8.0",
     "babel-plugin-transform-object-assign": "^6.8.0",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-plugin-transform-runtime": "^6.9.0",
     "babel-preset-es2015": "^6.3.13",
-    "babel-preset-react": "^6.3.13",
-    "babel-preset-react-hmre": "^1.1.0"
+    "babel-preset-react": "^6.3.13"
```

Comment:
The latest version is `6.16.0` so it's in the semver range.  
I will update it anyway, just so to say :)